tests/Aspire.Hosting.Tests/Azure/AzureBicepResourceTests.cs

```diff
@@ -458,8 +469,151 @@ public async Task PublishAsAzurePostgresFlexibleServer()
 
         var manifest = await ManifestUtils.GetManifest(postgres.Resource);
 
-        Assert.Equal("azure.bicep.v0", manifest["type"]?.ToString());
-        Assert.Equal("{postgres.secretOutputs.connectionString}", manifest["connectionString"]?.ToString());
+        var expectedManifest = """
+            {
+              "type": "azure.bicep.v0",
+              "connectionString": "{postgres.secretOutputs.connectionString}",
+              "path": "aspire.hosting.azure.bicep.postgres.bicep",
+              "params": {
+                "serverName": "postgres",
+                "keyVaultName": "",
+                "administratorLogin": "{usr.value}",
+                "administratorLoginPassword": "{pwd.value}",
+                "databases": [
+                  "db"
+                ]
+              },
+              "inputs": {
+                "password": {
```

Comment:
@davidfowl - I decided to leave this "password" input in the annotations, even if nothing in the resource references it. My reasoning was because if we start using `InputReference` in more places, removing it is going to cause problems. So it is better to leave it in, and not cause problems when we start enabling more scenarios.